src/react/src/routes/Router.js

```diff
@@ -3,6 +3,7 @@ import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import Homepage from './Homepage/Homepage';
 import SparkButtonDocs from './SparkButtonDocs/SparkButtonDocs';
 import SprkLinkDocs from './SprkLinkDocs/SprkLinkDocs';
+import SprkDividers from './SprkDividersDocs/SprkDividersDocs';
```

Comment:
I think these should be called "SprkDivider" (singular) and "SprkDividerDocs" to match the other component names.

Reply:
Updated!